plugins/modules/xbps.py

```diff
@@ -308,6 +377,9 @@ def main():
             upgrade=dict(default=False, type='bool'),
             update_cache=dict(default=True, type='bool'),
             upgrade_xbps=dict(default=True, type='bool'),
+            root=dict(type='str'),
```

Comment:
If you change it to path, that change should be applied to both this line and the docs above.